src/main/java/de/tum/in/www1/artemis/web/rest/ExerciseResource.java

```diff
@@ -401,4 +401,22 @@ private StatsForInstructorDashboardDTO populateCommonStatistics(Exercise exercis
 
         return ResponseUtil.wrapOrNotFound(Optional.of(exercise));
     }
+
+    /**
+     * GET /exercises/:exerciseId/toggle-second-correction
+     *
+     * @param exerciseId the exerciseId of the exercise to get the repos from
+     * @return the ResponseEntity with status 200 (OK) and with body the exercise, or with status 404 (Not Found)
+     */
+    @GetMapping(value = "/exercises/{exerciseId}/toggle-second-correction")
```

Comment:
A Get mapping seems wrong here as it breaks the HTTP contract for GET. PUT seems to be the most appropiate here

![image](https://user-images.githubusercontent.com/29718932/106144800-ff2a6080-6174-11eb-9da0-ee12ab09a28a.png)

Reply:
Nice catch, done 👍